scripts/plot_summary.py

```diff
@@ -385,9 +385,10 @@ def plot_carbon_budget_distribution():
     ax1.set_xlim([1990,snakemake.config['scenario']['planning_horizons'][-1]+1])
 
     path_cb = snakemake.config['results_dir'] + snakemake.config['run'] + '/csvs/'
-    countries=pd.read_csv(path_cb + 'countries.csv',  index_col=1)
+    pop_layout = pd.read_csv(snakemake.input.clustered_pop_layout, index_col=0)
+    countries=pd.read_csv(snakemake.input.country_codes, index_col=1)
```

Comment:
Long term, I would like to get rid of this csv file.

Reply:
I would keep it for now because it is handy to become all the countries. Maybe in the long term we can have in the make_summary script one function saving all the modelled countries